runner/server/handle.go

```diff
@@ -80,6 +80,35 @@ func (s *Server) handleServiceRequest() http.HandlerFunc {
 	}
 }
 
+func (s *Server) HandleApplyService() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		defer r.Body.Close()
+
+		// Verify token
+		_, err := s.auth.VerifyToken(utils.GetToken(r))
+		if err != nil {
+			logrus.Errorf("Failed to apply service - %s", err.Error())
+			utils.SendErrorResponse(w, r, http.StatusUnauthorized, err)
+			return
+		}
+
+		req := new(model.ServiceRequest)
+		json.NewDecoder(r.Body).Decode(req)
```

Comment:
The will be an event document if I'm not wrong. The data of the document will contain the payload. Refer: https://github.com/spaceuptech/space-cloud/blob/master/modules/eventing/handle_staged.go#L88